_sass/_show.scss

```diff
@@ -0,0 +1,193 @@
+.show {
+  --_show--gutter: var(--show--gutter, var(--content--gutter, 4rem));
+  --_show--column-gap: var(--show--column-gap, var(--content---column-gap, 3rem));
+  --_show--row-gap: var(--show--row-gap, var(--content---row-gap, 2rem));
+  --_show--padding-y: var(--show--padding-y, var(--content--padding-y, 3rem));
+  --_show--color: var(--show--color, #{$yellow});
+  --_show--bg--color--start: var(--show--bg--color--start, #{$white});
+  --_show--bg--color--end: var(--show--bg--color--end, color-mix(in oklab, var(--_show--color) 10%, var(--_show--bg--color--start)));
+
```

Comment:
Generally, I think there's too much space between the Title and the content. I would say, go ahead and remove the bottom margin on the `<h2>`, since the grid-gap is creating that space.

Reply:
While in the `h2` I think it could use some more line-height. `line-height: 1.5;`

Reply:
Great! thanks